mocks/handlers/aws/idm.ts

```diff
@@ -0,0 +1,30 @@
+import { http, HttpResponse } from "msw";
+
+const defaultIDMHandler = http.get(
+  "https://dimauthzendpoint.com/api/v1/authz/id/all?userId=:id",
```

Comment:
You can leave off `?userId=:id` it will still catch it and everything else will be the same. That way it will also catch anything with additional query params